src/test/java/seedu/address/logic/commands/AddStudentToClassCommandIntegrationTest.java

```diff
@@ -0,0 +1,107 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.logic.commands.CommandTestUtil.VALID_MODULE_AMY;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_MODULE_BOB;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_TUTORIAL_AMY;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_TUTORIAL_BOB;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandFailure;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.testutil.TypicalPersons.getTypicalAddressBook;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.addstudenttoclasscommands.AddStudentToClassByEmailCommand;
+import seedu.address.logic.commands.addstudenttoclasscommands.AddStudentToClassByIdCommand;
+import seedu.address.logic.messages.ModuleMessages;
+import seedu.address.logic.messages.PersonMessages;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.module.ModuleCode;
+import seedu.address.model.module.TutorialClass;
+import seedu.address.model.person.Person;
+import seedu.address.testutil.PersonBuilder;
+
+/**
+ * Contains integration tests (interaction with the Model) for
+ * {@code AddCommand}.
+ */
+public class AddStudentToClassCommandIntegrationTest {
+
+    private Model model;
+
+    @BeforeEach
+    public void setUp() {
+        model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+        ModuleCode newModule = new ModuleCode(VALID_MODULE_AMY);
+        model.addModule(newModule);
+        TutorialClass newTutorialClass = new TutorialClass(VALID_TUTORIAL_AMY);
+        newModule.addTutorialClass(newTutorialClass);
+        Person validPerson = new PersonBuilder().build();
+        model.addPerson(validPerson);
+    }
+
+    @Test
+    public void execute_addStudentToClassById_success() {
+        Model expectedModel = new ModelManager(model.getAddressBook(), new UserPrefs());
+
+        ModuleCode newModule = new ModuleCode(VALID_MODULE_AMY);
+        expectedModel.addModule(newModule);
+        TutorialClass newTutorialClass = new TutorialClass(VALID_TUTORIAL_AMY);
+        newModule.addTutorialClass(newTutorialClass);
+        Person validPerson = new PersonBuilder().build();
+
+        // Attempt to delete the student
```

Comment:
Should the comment be attempt to add the student instead?

Reply:
Good eye! Thanks for spotting that :)